test/e2e/app/run.go

```diff
@@ -121,6 +121,8 @@ func E2ETest(ctx context.Context, def Definition, cfg E2ETestConfig, secrets age
 
 	stopReceiptMonitor := StartMonitoringReceipts(ctx, def)
 
+	stopValidatorUpdates := StartValidatorUpdates(ctx, def)
```

Comment:
Should we run `stopValidatorUpdates` using defer? Otherwise, if any of the following step errors and we return an error this will not be executed.

Reply:
this is e2e test, it doesn't matter. as soon as anything fails, the test exists